resources/views/auth/login.blade.php

```diff
@@ -25,7 +25,7 @@
                 <div class="form-group">
                     <label for="login_id" class="sr-only">学籍番号・連絡先メールアドレス</label>
```

Comment:
矢継ぎ早に申し訳ありません！

こちらも修正お願いしたいです！

```suggestion
                    <label for="login_id" class="sr-only">学籍番号または連絡先メールアドレス</label>
```

Reply:
> ログインフォームのユーザー名入力欄の placeholder は「学籍番号または連絡先メールアドレス」と表示する。

`placeholder`とありましたのでその様にしました。その様でしたらまた該当する部分がありますので修正いたします。

Reply:
すみません、label の修正のほうは issue に書き忘れていました、、

issue に書いていない内容であっても、「こうしたほうが良いのでは？」などあれば、ついでに修正してもらっても OK です！

Reply:
了解です